### ISSUE TITLE
Example seems to imply that min_duration counts parking time only, but spec says it counts whole Charging Session

### ISSUE THREAD

**Issue by reinierl** [ocpi/ocpi#434]
The description of the example in the file `tariff_7_first_hour_kwh_free.json` implies that this tariff gives the first hour of parking free:

> In this example, we have the following scenario:
>  *  The first hour of parking time is free.

It seems to do so using a `min_duration` restriction in the TariffElement:

```
...
        "elements": [{
                "price_components": [{
                        "type": "PARKING_TIME",
                        "price": 0.0,
                        "step_size": 60
                }],
                "restrictions": {
                        "min_duration": 0,
                        "max_duration": 3600
                }
        }, {
...
```

The specification however says that `min_duration` applies to duration from the beginning of the Charging Session, not from the beginning of the Charging Period:

> Minimum duration in seconds the Charging Session MUST last (inclusive). When the duration of a Charging Session is longer than the defined value, this TariffElement is or becomes active. Before that moment, this TariffElement is not yet active.

Therefore applying the example Tariff JSON to the scenario described does not give the result that the example text describes.

tl;dr: The example is wrong.

**Comment by reinierl** [comment]
I think that for OCPI 3.0, we can and should go for the second solution.

For OCPI 2.2.1, we can't change the specification, we can only clarify it. So I think that, if I had to decide on the spot here, I would just remove the example.

**Comment by reinierl** [comment]
We decided in the OCPI Development Working Group call at 2023-05-11 that we indeed remove the example from OCPI 2.2.1 in the next documentation release. We will make loitering fees possible again in OCPI 3.0.

**Comment by paulintrognon** [comment]
Just a quick note to mention that GIREVE (the primary interoperability actor in France) utilizes `min_duration` solely for parking duration when linked with a PARKING_TIME element.

**Comment by Hamza-nabil** [comment]
Thank you @paulintrognon  for the note!
I couldn’t find any reference to min_duration for parking in [GIREVE’s docs ](https://github.com/CNX-GIREVE/GIREVE_Tech_OCPI_V2.1.1/ )

Is there another source I missed?
